client/my-sites/plans-v2/translated-copy.tsx

```diff
@@ -0,0 +1,203 @@
+				),
+				buttonLabel: translate( 'Get Jetpack Complete' ),
+			};
+		case PRODUCT_JETPACK_CRM:
+		case PRODUCT_JETPACK_CRM_MONTHLY:
+			return {
+				displayName: translate( 'Jetpack CRM' ),
+				shortName: translate( 'CRM', {
+					comment: 'Short name of the Jetpack CRM',
+				} ),
+				tagline: translate( 'Manage contacts effortlessly' ),
+				description: translate(
+					'The most simple and powerful WordPress CRM. Improve customer relationships and increase profits.'
+				),
+				buttonLabel: translate( 'Get CRM' ),
+			};
+		case PRODUCT_JETPACK_SCAN:
+		case PRODUCT_JETPACK_SCAN_MONTHLY:
+			return {
+				displayName: translate( 'Jetpack Scan' ),
+				shortName: translate( 'Scan', {
+					comment: 'Short name of Jetpack Scan',
+				} ),
+				tagline: translate( 'Protect your site' ),
+				description: translate(
+					'Automatic scanning and one-click fixes keep your site one step ahead of security threats.'
+				),
+				buttonLabel: translate( 'Get Scan' ),
+			};
+		case OPTIONS_JETPACK_SECURITY:
+		case OPTIONS_JETPACK_SECURITY_MONTHLY:
+			return {
+				displayName: translate( 'Jetpack Security' ),
+				shortName: translate( 'Security', {
+					comment: 'Short name of the Jetpack Security generic plan',
+				} ),
+				tagline: translate( 'Comprehensive WordPress protection' ),
+				description: translate(
+					'Enjoy the peace of mind of complete site security. ' +
+						'Easy-to-use, powerful security tools guard your site, so you can focus on your business. ' +
+						'{{em}}Available options: Real-time or Daily.{{/em}}',
+					{ components: { em: <em /> } }
+				),
+				buttonLabel: translate( 'Get Jetpack Security' ),
+			};
+		case PLAN_JETPACK_SECURITY_DAILY:
+		case PLAN_JETPACK_SECURITY_DAILY_MONTHLY:
+			return {
+				displayName: translate( 'Security {{em}}Daily{{/em}}', { components: { em: <em /> } } ),
+				shortName: translate( 'Security Daily', {
+					comment: 'Short name of the Jetpack Security Daily plan',
+				} ),
+				tagline: translate( 'Comprehensive WordPress protection' ),
+				description: translate(
+					'Enjoy the peace of mind of complete site protection. ' +
+						'Great for brochure sites, restaurants, blogs, and resume sites.'
+				),
+				buttonLabel: translate( 'Get Security {{em}}Daily{{/em}}', { components: { em: <em /> } } ),
+			};
+		case PLAN_JETPACK_SECURITY_REALTIME:
+		case PLAN_JETPACK_SECURITY_REALTIME_MONTHLY:
+			return {
+				displayName: translate( 'Security {{em}}Real-time{{/em}}', { components: { em: <em /> } } ),
+				shortName: translate( 'Security Real-time', {
+					comment: 'Short name of the Jetpack Security Real-time plan',
+				} ),
+				tagline: translate( 'Comprehensive WordPress protection' ),
+				description: translate(
+					'Additional security for sites with 24/7 activity. ' +
+						'Recommended for eCommerce stores, news organizations, and online forums.'
+				),
+				buttonLabel: translate( 'Get Security {{em}}Real-time{{/em}}', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Security {{em}}Real-time{{/em}}' )` *ES Score: 9*